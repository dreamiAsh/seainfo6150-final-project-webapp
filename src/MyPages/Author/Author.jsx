import React, {useEffect, useState} from "react";
import styles from "./Author.module.css";
import RecipeList from "../../Components/RecipeList/RecipeList"

const Author = (props) => {

    return (
        <div>
            <p className={styles.author}>Author:   <span className={styles.inAuthor}>{props.author}</span></p>
            <p className={styles.authorInfo}>Self-introduction:   <span className={styles.inAuthorInfo}>{props.authorInfo}</span></p>
            <p className={styles.guess}>Guess you may like</p>
            <RecipeList/>
        </div>
    )
}

export default Author
