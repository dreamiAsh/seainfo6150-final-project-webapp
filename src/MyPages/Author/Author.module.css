.author{
    color: orange;
    font-weight: bold;
    margin: 100px 0 20px 13%;
}
.authorInfo{
    color: orange;
    font-weight: bold;
    margin: 0 0 20px 13%;
}
.inAuthor{
    color: black;
    font-weight: normal;
}
.inAuthorInfo{
    color: black;
    font-weight: normal;
}
.guess{
    font-weight: bold;
    font-size: 22px;
    margin: 90px 0 20px 11%;
}