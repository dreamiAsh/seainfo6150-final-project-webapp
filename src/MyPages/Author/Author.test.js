import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Author from "./Author.jsx";

it("renders the Author component correctly", () => {
  const { container } = render(<Author />);
  expect(container).toMatchSnapshot();
});