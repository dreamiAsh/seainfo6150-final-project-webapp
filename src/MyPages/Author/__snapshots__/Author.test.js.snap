// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Author component correctly 1`] = `
<div>
  <div>
    <p
      class="author"
    >
      Author:   
      <span
        class="inAuthor"
      />
    </p>
    <p
      class="authorInfo"
    >
      Self-introduction:   
      <span
        class="inAuthorInfo"
      />
    </p>
    <p
      class="guess"
    >
      Guess you may like
    </p>
    <ul
      class="recipeList"
    />
  </div>
</div>
`;
